seed/campaigns.ts

```diff
@@ -1,59 +1,143 @@
-import { Prisma, PrismaClient, User } from '@prisma/client'
+import {
+  CampaignUpdateHistoryType,
+  Prisma,
+  PrismaClient,
+  User,
+} from '@prisma/client'
 import { campaignFactory } from './factories/campaign.factory'
 import { campaignUpdateHistoryFactory } from './factories/campaignUpdateHistory.factory'
 import { userFactory } from './factories/user.factory'
 import { pathToVictoryFactory } from './factories/pathToVictory.factory'
 import { buildSlug } from 'src/shared/util/slug.util'
 import { getFullName } from 'src/users/util/users.util'
-
-const NUM_CAMPAIGNS = 20
+import { Campaign } from '@prisma/client'
+import fixedCampaigns from './fixedCampaigns.json'
+import 'prisma/schema/pathToVictory.jsonTypes'
+const NUM_GENERATED_CAMPAIGNS = 40
 const NUM_UPDATE_HISTORY = 3
+const FIXED_CAMPAIGNS: Partial<Campaign>[] = fixedCampaigns
+
+type CampaignUpdateHistory = {
+  id: number
+  createdAt: Date
+  updatedAt: Date
+  userId: number
+  campaignId: number
+  type: CampaignUpdateHistoryType
+  quantity: number
+}
+
+type FakeP2V = {
+  id: number
+  createdAt: Date
+  updatedAt: Date
+  campaignId: number
+  data: PrismaJson.PathToVictoryData
+}
 
 export default async function seedCampaigns(
   prisma: PrismaClient,
   existingUsers: User[],
 ) {
-  const fakeP2Vs: any[] = []
-  const fakeUpdateHistory: any[] = []
-
+  const fakeP2Vs: FakeP2V[] = []
```

Comment:
I don't see you using this anywhere other than filling it up?

<img width="585" alt="image" src="https://github.com/user-attachments/assets/200ab0aa-841c-4de8-8af4-7db7ea778c77" />

Seems like you could just get rid of it here as well as from the return from `createCampaignAndUser` 🤔